pinot-connectors/pinot-spark-common/src/main/scala/org/apache/pinot/connector/spark/common/PinotClusterClient.scala

```diff
@@ -32,17 +32,23 @@ import com.fasterxml.jackson.annotation.JsonIgnoreProperties
  * PinotCusterClient reads metadata from Pinot controller.
  */
 private[pinot] object PinotClusterClient extends Logging {
-  private val TABLE_SCHEMA_TEMPLATE = "http://%s/tables/%s/schema"
-  private val TABLE_BROKER_INSTANCES_TEMPLATE = "http://%s/v2/brokers/tables/%s"
-  private val TIME_BOUNDARY_TEMPLATE = "http://%s/debug/timeBoundary/%s"
-  private val ROUTING_TABLE_TEMPLATE = "http://%s/debug/routingTable/sql?query=%s"
-  private val INSTANCES_API_TEMPLATE = "http://%s/instances/%s"
+  private val TABLE_SCHEMA_TEMPLATE = "%s://%s/tables/%s/schema"
```

Comment:
Seems like we are replacing http with https, should we keep both?

Reply:
yes, the underlying scheme is set by `useHttps`